src/rust/ide/src/view/project_name.rs

```diff
@@ -4,19 +4,60 @@ use crate::prelude::*;
 
 use crate::controller;
 
+use enso_frp as frp;
+use ensogl::data::color;
 use ensogl::display;
-use ensogl::display::world::World;
+use ensogl::display::Attribute;
+use ensogl::display::Buffer;
 use ensogl::display::object::ObjectOps;
+use ensogl::display::scene::Scene;
 use ensogl::display::shape::text::glyph::font;
 use ensogl::display::shape::text::text_field::TextField;
 use ensogl::display::shape::text::text_field::TextFieldProperties;
-use ensogl::data::color;
-
+use ensogl::display::shape::*;
+use ensogl::display::Sprite;
+use ensogl::display::world::World;
+use ensogl::gui::component::Animation;
+use ensogl::gui::component;
+use logger::enabled::Logger;
+use logger::AnyLogger;
 use nalgebra::Vector2;
 
+
+
+// =================
+// === Constants ===
+// =================
+
+const HIGHLIGHTED_TEXT_COLOR : color::Rgba = color::Rgba::new(1.0,1.0,1.0,1.0);
+const DARK_GRAY_TEXT_COLOR   : color::Rgba = color::Rgba::new(0.6,0.6,0.6,1.0);
```

Comment:
`(0.6,0.6,0.6,1.0)` doesnt play nice with bg. Bg is a little yellowish. Shoulduse alpha instead.